src/css/research.css

```diff
@@ -0,0 +1,856 @@
+}
+
+/* add a labs label to a title in a card */
+body.research .header-label-container > div.labels {
+  display: inline-flex;
+  margin-left: auto;
+}
+
+body.research .cards .sect2 > a,
+body.research .cards:not(.selectable) .sect2 > a,
+body.research .widget:not(.video) {
+  /* box-shadow: 0 2px 4px -1px rgba(0, 0, 0, 0.1), 0 1px 2px -1px rgba(0, 0, 0, 0.06); */
+  border: none;
+  border-radius: 1rem;
+}
+
+body.research .cards .sect2 > a {
+  color: var(--neutral-color);
+  font-weight: var(--font-weight-normal);
+  text-decoration: none;
+}
+
+body.research.color-docs .sect2 {
+  color: rgba(var(--colors-neutral-10));
+  background: var(--color-docs);
+}
+
+body.research.color-docs .cards .sect2 h3,
+body.research.color-docs .cards .sect2 .description {
+  color: rgba(var(--colors-neutral-10));
+}
+
+body.research .cards .sect2 .paragraph {
+  margin: 0;
+}
+
+/* body.research:not(.create-applications) .sect2 {
+  min-height: 200px;
+} */
+
+body.research .cards .sect2 a > h3,
+body.research .cards .sect2 a > div {
+  margin: 0;
+  padding: 0;
+}
+
+body.research .highlights .sectionbody > div,
+body.research .sect2 a > div,
+body.research .sect2 a > h3 {
+  display: flow;
+}
+
+body.research .cards .sect2 .icon,
+body.research .widget .icon {
+  padding: 0;
+  margin: 0;
+  width: 100%;
+}
+
+body.research .cards .sect2 .icon,
+body.research .widget.banner .icon {
+  order: 1;
+  padding: 0;
+  margin: 0;
+  width: 100%;
+}
+
+body.research .cards .sect2 .icon p {
+  display: flex;
+  width: -webkit-fill-available;
+  justify-content: space-between;
+}
+
+body.research .cards .sect2 .icon p div.labels {
+  display: flex;
+  flex-direction: row;
+  gap: 0.25rem;
+  height: min-content;
+}
+
+body.research .cards .icon span {
+  display: flex;
+}
+
+/* For the 1st, 3rd etc highlights, move the image to the right */
+body.research .widget.highlights:nth-of-type(odd) .icon {
+  order: 1;
+}
+
+/* We can use this rule to apply styles to the 2nd, 4th highlights etc widget */
+/* body.research .widget.highlights:nth-of-type(even) .openblock {
+  margin-right:40px;
+} */
+
+body.research body.research .cards .sect2 .icon img {
+  width: 4rem;
+  height: 4rem;
+  padding: 0;
+}
+
+body.research .cards .sect2 .icon.icon-l img {
+  width: 6rem;
+  height: 6rem;
+  margin-bottom: 9px;
+}
+
+body.research .lists .sect2 .icon img,
+body.research .icon-s .sect2 .icon img {
+  width: 3rem;
+  height: 3rem;
+  padding: 0;
+}
+
+body.research .lists .sect2 .icon img path {
+  color: rgba(var(--colors-baltic-30));
+  fill: rgba(var(--colors-baltic-30));
+}
+
+body.research .cards .sect2 h3,
+body.research .lists .sect2 h3 {
+  /* flex-grow: 1; */
+  order: 2;
+  margin: 0;
+  font-weight: var(--font-weight-semibold);
+  width: -webkit-fill-available;
+}
+
+body.research .cards .sect2 h3 .anchor::before {
+  display: none;
+}
+
+body.research .cards .sect2 .paragraph:not(.icon) {
+  font-size: var(--font-size-body-medium);
```

Comment:
I don't know if we want to use this smaller font size for paragraphs of text. I can see that it results in two evenly sized columns, which is nice, but I don't like it all that much side-by-side with a different font size. I'm also not sure if there are accessibility issues at this size.

Reply:
Hmmm that's how it is in docs-ndl, I just brought it to the research layout

Reply:
Fair point! We do use it in cards as well as lists in docs-ndl pages. I might be arguing from personal preference but I think it would be better here if the font size was the same in the left and right columns, and if we are picking a size I would go for the `1rem`.

Reply:
In the concept given by the designers, there's a difference in the size of the fonts in each column too (left is 16px and right is 14px). I don't have a personal preference, I'm just highlighting that the difference exists in the concept too, so we might get feedback on that

Reply:
We don't have to take their choice, but I'll leave it up to you. I think we should to find good justifications for changing a font size but maybe it's not a major consideration here.